teps/0003-tekton-catalog-organization.md

```diff
@@ -408,7 +408,60 @@ For example (with Task and Pipelines):
       /samples/dummy-go-release.yaml |
 ```
 
-An example of a *freezed* catalog migrated to this layout is available
+
+#### Example of a resource from catalog
+
+* An example of `git-clone` task from the catalog: [Git-Clone](https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1)
+
+* Fields added to the yaml file
+
+```yaml
+  labels:
+    app.kubernetes.io/version: "0.1"           👈 MUST: version of the resource
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"  👈 MUST: version of pipeline
+    tekton.dev/tags: git                       👈 Optional: Comma separated list of tags
+    tekton.dev/displayName: "git clone"        👈 Optional: Display name of the task
+spec:
+  description: >-
+    These Tasks are Git tasks to work with repositories used by other tasks
```

Comment:
Since this is an example, how about we provide a better summary for others to inspire from ;-) 

E.g.

> git-clone clones git repositories (usually to a workspace) for use with other tasks in a pipeline.

> git-clone clones a git repository pointed by the param - URL into an output workspace. By default, the repository will be cloned into the root of the workspace. The location can be changed by setting the param -  subdirectory.